cardano-cli/src/Cardano/CLI/EraBased/Run/Certificate.hs

```diff
@@ -266,31 +267,30 @@ runGovernanceRegistrationCertificate anyReg outfp =
 
 getStakeCredentialFromVerifier
   :: StakeVerifier
-  -> ExceptT ShelleyStakeAddressCmdError IO StakeCredential
```

Comment:
This returned `ShelleyStakeAddressCmdError` which makes re-use difficult.  The intent is to move this code to `Cardano.CLI.Types.Key`, but `Cardano.CLI.Types.Key` should never reference a command error because contains utility types and functions to be used by commands not the other way around.